Makefile

```diff
@@ -96,7 +96,7 @@ cache:
 
 database-build:
 	@echo "+\n++ Performing postgres build ...\n+"
-	@docker-compose build postgres
+	@docker-compose build postgres flyway
```

Comment:
What is this doing?

Reply:
Makes sure our flyway docker images are rebuilt everytime you rebuild the database so you can get the latest migrations.
Earlier we used to volume mount but i made some change a while back that broke that behaviour :/